components/shared/workCard/WorkCardWithCaption.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+
+import { cn } from "@/lib/helpers/helper.cn"
+import { displayCreators } from "@/lib/helpers/helper.creators"
+
+import WorkCard, { WorkCardProps } from "./WorkCard"
+
+type WorkCardWithCaptionProps = {
+  classNameCaption?: string
+} & WorkCardProps
+
+const WorkCardWithCaption = ({
+  work,
+  classNameCaption,
+  className,
+  stackPosition,
+  isWithTilt,
+}: WorkCardWithCaptionProps) => {
+  return (
+    <div className="block space-y-3 lg:space-y-5">
+      <WorkCard work={work} className={className} isWithTilt={isWithTilt} />
```

Comment:
className should overwrite styling on the root element of the component, not the workCard.

```suggestion
      <WorkCard work={work} isWithTilt={isWithTilt} />
```